client/src/components/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles({
+    root: {
+        border:' solid rgb(241, 238, 232) 1px',
+        borderRadius:'2%',
+        width:'50% ',
+        margin: '50px auto',
+        textAlign:'center',
+        backgroundColor:'rgb(241, 238, 232)',
+        padding:'30px',
+        '& form':{
+            '& h2':{
+            },
+            '& .textField':{
```

Comment:
I think it's more clear to have each new class name be on its own. Like:
root: {
   border, etc...
},
textField: {
   width, etc...
}